parsec/backend/postgresql/organization.py

```diff
@@ -61,15 +62,21 @@
 """
 )
 
-
 _q_get_stats = Q(
     f"""
 SELECT
     (
         SELECT COUNT(*)
         FROM user_
-        WHERE user_.organization = { q_organization_internal_id("$organization_id") }
+        WHERE user_.organization = { q_organization_internal_id("$organization_id") } AND
```

Comment:
We can use array and tuple to fetch profile and revocation date in a single query:
```sql
SELECT ARRAY(SELECT (revoked_on, profile) FROM user_ WHERE organization = 42);
```

This would allow us to crunch better stats:
```python
{
  ...
  "users": ...,
  "active_users": ...,
  "users_per_profile_detail": {
     "ADMINISTRATOR": {"active": ..., "revoked": ...},
     "STANDARD": {"active": ..., "revoked": ...},
     "OUTSIDER": {"active": ..., "revoked": ...},
  }
}
```